crates/oxide/src/extractor/pre_processors/test-fixtures/haml/dst-1.haml

```diff
@@ -0,0 +1,155 @@
+          :markdown
+            **PLUS!** You can utlize filters like _Markdown_, CoffeeScript,
+            Textile, and many more!
+
+       mt-8 xl:mt-0
+         flex flex-col xl:flex xl:flex-row w-full
+            doc_code css: "grow", code_css: "xl:pl-2 xl:pr-0 xl:rounded-r-none", language: "erb"  do
+            :escaped
+              <%   Ruby %>
+
+              <% 5 times do |i| %>
+                <% if i even? %>
+                  <p class "odd">Number <%  i %></p>
+                <% else %>
+                  <p class "even">Numero <%  i %></p>
+                <% end %>
+              <% end %>
+
+            doc_code css: "grow mt-8 xl:mt-0", pre_css: "xl:h-full", code_css: "xl:pl-0 xl:pr-2 xl:h-full xl:rounded-l-none"  do
+            :escaped
+              -   HAML
+
+              - 5 times do |i|
+                - if i even?
+                  %p odd Number \  i 
+                - else
+                  %p even Numero \  i 
+
+
+    %h3 mt-32 text-4xl font-bold
+      Build Your <i>OWN</i> Components
+
+    %p mt-2 text-xl
+      Can't find exactly what you need? No problem! Build your own components
+      with ease using our simple, flexible, and powerful DSL 
+
+     mt-8 xl:flex xl:space-x-8
+      %div
+          doc_code language: "ruby"  do
+          :escaped
+              app/components/application_component rb
+            class ApplicationComponent < LocoMotion::BaseComponent
+                Add your custom / shared component logic here!
+            end
+
+          doc_code language: "haml", css: "mt-8"  do
+          :escaped
+            -   app/components/character_component html haml
+              part :component  do
+                part :head 
+                part :body  do
+                  content
+                part :legs 
+
+      %div mt-8 xl:mt-0
+          doc_code language: "ruby"  do
+          :escaped
+              app/components/character_component rb
+            class CharacterComponent < ApplicationComponent
+              define_parts :head, :body, :legs
+
+              def before_render
+                set_tag_name :head, :h1 
+                add_css :head, "text-3xl font-bold" 
+
+                set_tag_name :body, :p 
+                add_stimulus_controller :body, "character-body" 
+                add_css :body, "text-lg" 
+
+                set_tag_name :legs, :footer 
+                add_css :legs, "text-sm" 
+              end
+            end
+
+    %h3 mt-24 text-center text-4xl font-bold italic
+      More Coming Soon!
+    %p mt-2 text-xl text-center
+      Keen an eye out as we'll be adding more components, guides, and
+      %br max-sm:hidden
+      suggested gems for you to build amazing Rails apps!
+     mt-4 text-center
+        daisy_button "😉 Get Started", css: "btn-primary text-xl",
+        right_icon: "arrow-right", target: "_blank",
+        class: "px-2.5"
+        href: "https://github com/profoundry-us/loco_motion locomotion-components"
```

Comment:
A bit silly, but the `.` and `#` characters are replaced because:

1. The `.` is not surrounded by numbers
2. The `#` is replaced because it's not inside of a bracket stack

While this does break the string, it doesn't matter from Tailwind's perspective and since we're not updating the real code, it's not an issue.